web/ui/react-app/src/pages/alerts/AlertContents.tsx

```diff
@@ -10,7 +10,7 @@ export interface Rule {
   alerts: Alert[];
   annotations: Record<string, string>;
   duration: number;
-  health: string;
+  health: RuleState;
```

Comment:
I'd prefer if we could only allow the actually valid values for `health` and `state` here. Maybe just get rid of the `export type RuleState = keyof RuleStatus<any>;` and do a direct `health: 'firing' | 'pending' | 'inactive'` here and `state: 'unknown' | 'err' | 'ok'` below? The alternative would be to have two separate enums / interfaces at the beginning and then joining them into one later on somehow.